custom_components/tuya_local/devices/breville_smartairconnect_purifier.yaml

```diff
@@ -0,0 +1,125 @@
+name: Breville Smart Air Connect Purifier
+products:
+  - id: tltdxtehttjbkjni
+primary_entity:
+  entity: fan
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: true
+          icon: "mdi:air-purifier"
+        - dps_val: false
+          icon: "mdi:air-purifier-off"
+    - id: 3
+      name: mode
+      type: string
+      mapping:
+        - dps_val: "manual"
+          value: "Manual"
+        - dps_val: "auto"
+          value: "Auto"
+    - id: 4
+      type: string
+      name: speed
+      mapping:
+        - dps_val: low
+          value: 33
+        - dps_val: mid
+          value: 66
+        - dps_val: high
+          value: 100
+secondary_entities:
+  - entity: sensor
+    name: PM2.5
+    icon: "mdi:air-filter"
+    class: pm25
+    dps:
+      - id: 2
+        name: sensor
+        class: measurement
+        type: integer
+        unit: µg/m3
+  - entity: select
+    name: Fan Mode
+    icon: "mdi:auto-mode"
+    category: config
+    dps:
+      - id: 3
+        type: string
+        name: option
+        mapping:
+          - dps_val: "auto"
+            value: "Auto"
+          - dps_val: "manual"
+            value: "Manual"
+  - entity: switch
+    name: Night mode
+    category: config
+    icon: "mdi:lightbulb-night"
+    dps:
+      - id: 8
+        name: switch
+        type: boolean
+  - entity: switch
+    name: Micro shield
+    category: config
+    icon: "mdi:shield-bug"
+    dps:
+      - id: 9
+        name: switch
+        type: boolean
+  - entity: sensor
+    name: Filter days left
+    category: diagnostic
```

Comment:
Suggest adding `class: duration` . unit (below) should be `d` (see https://github.com/home-assistant/core/blob/dev/homeassistant/const.py#L595)

Reply:
Done, the unit is now `d`.